.github/workflows/scanner-versioned-definitions-update.yaml

```diff
@@ -166,8 +166,11 @@ jobs:
         echo "STACKROX_NVD_ZIP_PATH=$path" >> "$GITHUB_ENV"
 
     - name: Run Updater (single bundle)
+      if: ${{ env.SCANNER_BUNDLE_VERSION != 'dev' }}
       run: |
-        scanner/bin/updater export --manual-url "${{ needs.prepare-environment.outputs.manual_url }}" "definitions/${{ env.SCANNER_BUNDLE_VERSION }}"
+        if [[ "${SCANNER_BUNDLE_VERSION//[^0-9.]/}" < "4.6.0" ]]; then
+          scanner/bin/updater export --manual-url "${{ needs.prepare-environment.outputs.manual_url }}" "definitions/${SCANNER_BUNDLE_VERSION}"
+        fi
```

Comment:
This doesn't look like would work well with two-digit semantic version streams (like `4.6.10`).

I suggest you look into using `case $SCANNER_BUNDLE_VERSION in` for this.

It might also be worth echoing we are not building the single bundle for anything other than `4.5.*` or `4.4.*`.

Reply:
Actually, I just realized `SCANNER_BUNDLE_VERSION` will be `v1`, `v2`, etc. Maybe what we want is to only run this for v1?

Reply:
Yeah, this is definitely wrong. Good catch.